src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.netcoreapp1.1.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.NetCore.Extensions;
+
+namespace System.IO.Compression.Tests
+{
+    public partial class ZipFileTest_ConvenienceMethods : ZipFileTestBase
+    {
+        [Fact]
+        public void ExtractToDirectoryOverwrite()
+        {
+            string zipFileName = zfile("normal.zip");
+            string folderName = zfolder("normal");
+
+            using (var tempFolder = new TempDirectory(GetTestFilePath()))
+            {
+                ZipFile.ExtractToDirectory(zipFileName, tempFolder.Path, overwriteFiles: false);
+                Assert.Throws<UnauthorizedAccessException>(() => ZipFile.ExtractToDirectory(zipFileName, tempFolder.Path /* default false */));
```

Comment:
On these exceptions, you'll need to check for IOException rather than UnauthorizedAccessException